themes/default/layouts/page/product.html

```diff
@@ -41,19 +35,19 @@ <h2>{{ .Params.overview.title }}</h2>
 
                 <ul id="product-carousel-headers" class="product-carousel-headers">
                     <li class="" data-product-carousel-type="build">
-                        <p class="my-0 text-black"><a href="#build" class="text-black">Build</a></p>
+                        <p class="my-0 text-black"><a href="#build" class="text-black">Languages</a></p>
```

Comment:
for this whole section, could we just use what's currently on the /product/ page? it should be the same graphics that's currently on /product/ where it says Build/Deploy/Manage.

Reply:
sure. i was trying to match what was in the design mock ups. i will revert this section back to how the page currently is.